packages/builder/src/stores/builder/screens.ts

```diff
@@ -138,13 +148,6 @@ export class ScreenStore extends BudiStore {
       }
 
       const definition = componentStore.getDefinition(component._component)
-      // Reset whitelist for direct children
-      legalDirectChildren = []
```

Comment:
I cannot see this being referenced anywhere. I assume is safe to remove. Could you confirm, @aptkingston ?